prokube-ai/kubeflow-examples
# Llama2 chat UI

In this PR I added an image for the chatbot UI for our llama2-chat deployment. It obviously needs the model API to be running, so it makes sense to first merge the other llama2 PR and then go ahead with this one.

How to test?

Checkout [this PR](https://github.com/prokube-ai/kubeflow-examples/pull/5), and make sure the kserve inference service for llama2-chat is running. Also, make sure your machine can reach this API. Then follow the instructions in the README to get the Dash app running as a Kubeflow notebook.
